src/ts/core/features/fuzzy_date.ts

```diff
@@ -1,17 +1,42 @@
 import * as chrono from 'chrono-node'
+import {browser} from 'webextension-polyfill-ts'
 import {afterClosingBrackets} from '../common/brackets'
 import {RoamDate} from '../roam/date'
 import {RoamNode, Selection} from '../roam/roam-node'
 import {Roam} from '../roam/roam'
 import {NodeWithDate} from '../roam/date/withDate'
 
-export const guard = ';'
-const dateContainerExpr = /;(.{3,}?);/gm
+import {Feature, Settings} from '../settings'
+
+export const config: Feature = {
+    id: 'fuzzy-date',
+    name: 'Fuzzy Date',
+    enabledByDefault: true,
+    settings: [{type: 'string', id: 'guard', initValue: ';', label: 'Guard symbol'}],
+}
+
+Settings.isActive(config.id).then(active => {
+    if (active) {
+        Settings.get(config.id, 'guard').then((value: string) => {
+            registerEventListener(value)
+        })
+    }
+})
+
+browser.runtime.onMessage.addListener(async message => {
```

Comment:
this handles enabling on toggle, but not disabling - see livePreview for example of disabling